README.md

```diff
@@ -21,6 +21,7 @@ Send a PR and help other people as well to find great companies to work for 💪
 1. [Setup](#setup) the repo
 2. Navigate to `content/companies`
 3. Create a file as `index-company_name.md`
+4. Follow the markdown template in `content/companies/02-gatsby.md`
```

Comment:
Okay so this confusing since it’s not really a template. But I like the Idea. So can you create a  file as `src/template.mdx` and put the things which makes a template there and  then add a link to that file in README?